### PR TITLE
[BAD-272][CLEANUP] Remove spark.yarn.jar param, should be set in spark config instead

### PR DESCRIPTION
**Pull request by pavel-sakun**
@mattyb149 please take a look

### CODE REVIEW

**Comment by kcruzada** [comment]
@mattyb149 can you process this?  code freeze is tomorrow for 5.4.

File: assembly/package-res/samples/jobs/Spark Submit/Spark submit.kjb [hunk 1]
```diff
@@ -29,13 +29,15 @@
...
-      <attribute><code>PORT_NUMBER</code><attribute>50000</attribute></attribute>
+      <attribute><code>PORT_NUMBER</code><attribute>50006</attribute></attribute>
       <attribute><code>PRESERVE_RESERVED_WORD_CASE</code><attribute>Y</attribute></attribute>
       <attribute><code>SUPPORTS_BOOLEAN_DATA_TYPE</code><attribute>Y</attribute></attribute>
       <attribute><code>SUPPORTS_TIMESTAMP_DATA_TYPE</code><attribute>Y</attribute></attribute>
     </attributes>
   </connection>
     <slaveservers>
+         <slaveserver><name>DI</name><hostname>localhost</hostname><port>9080</port><webAppName>pentaho-di</webAppName><username>admin</username><password>Encrypted 2be98afc86aa7f2e4bb18bd63c99dbdde</password><proxy_hostname/><proxy_port/><non_proxy_hosts/><master>Y</master><sslMode>N</sslMode></slaveserver>
+         <slaveserver><name>remoteDI</name><hostname>10.6.117.134</hostname><port>9080</port><webAppName>pentaho-di</webAppName><username>admin</username><password>Encrypted 2be98afc86aa7f2e4bb18bd63c99dbdde</password><proxy_hostname/><proxy_port/><non_proxy_hosts/><master>Y</master><sslMode>N</sslMode></slaveserver>
```
Comment (mattyb149) [inline assembly/package-res/samples/jobs/Spark Submit/Spark submit.kjb]: These slaveserver entries need to be removed

**Comment by kcruzada** [comment]
@pavel-sakun please remove the entries. Thanks!

**Comment by pavel-sakun** [comment]
@kcruzada @mattyb149 done.

**Pull request merged**